packages/accordions/src/styled/timeline/StyledSeparator.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'timeline.content.separator';
+
+interface IStyledSeparator {
+  surfaceColor?: string;
+}
+
+/**
+ * 1. override CSS bedrock
+ */
+export const StyledSeparator = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledSeparator>`
+  position: relative;
+  padding: ${props => `${props.theme.space.base * 5}px ${props.theme.space.base}px`};
+
+  svg {
+    box-sizing: content-box; /* [1] */
+  }
+
+  svg,
+  img {
+    position: relative;
+    z-index: 2;
+    background: ${props => props.surfaceColor || props.theme.colors.background};
+    padding: ${props => props.theme.space.base}px 0;
+    color: ${props => getColor('neutralHue', 600, props.theme)};
+  }
+
+  &::after {
+    position: absolute;
+    left: calc(50% - 0.5px);
```

Comment:
Looks like you're getting subpixel issues here with the use of `.5px` looking at the deployed storybook the line is slightly off center. In fact if you remove the calc and just do 50% it doesn't actually move so the browser is ignoring the subpixel shift all together.

To actually make this truly center you'd need to make the circle an uneven number width e.g. 13px then you can do `left: (50% - 1px)` to have true center, I can't access the design assets so not sure if they're doing subpixel alignment or not.

Reply:
I'm not seeing where the subpixel issues are. It looks like applying `left: 50%;` would cause a centering issue:

Currently, looks centered:
<img width="815" alt="Screen Shot 2021-06-10 at 11 22 14 AM" src="https://user-images.githubusercontent.com/1811365/121577214-296f1300-c9de-11eb-9b3b-18dc528dd9b1.png">

With `left: 50%;` it no longer looks centered:
<img width="814" alt="Screen Shot 2021-06-10 at 11 21 47 AM" src="https://user-images.githubusercontent.com/1811365/121577208-283de600-c9de-11eb-86c0-daa5f44a1c27.png">

Reply:
I think Garden needs to support centering for all sized icons. Garden's `svg-icons` (all even sizing) are used here. But these icons can be replaced with any valid React node including images, emojis, etc.

With an uneven size of `13px` and `left: calc(50% - 0.5)` it looks centered:
<img width="816" alt="Screen Shot 2021-06-10 at 11 38 33 AM" src="https://user-images.githubusercontent.com/1811365/121579586-b4e9a380-c9e0-11eb-8611-5d7f1904b5b7.png">

With an uneven size of `13px` and `left: calc(50% - 1)` it looks not centered:
<img width="815" alt="Screen Shot 2021-06-10 at 11 37 27 AM" src="https://user-images.githubusercontent.com/1811365/121579575-b1561c80-c9e0-11eb-97eb-d45d0f7b00b5.png">

Reply:
I noticed that the screenshots I posted in the pull request description is out-of-date. I think the Netlify deploy (click "View Deployment") should be up-to-date and have a centered line.

Reply:
I am noticing pixel issues here — but only sometimes — I think it can occur when the viewport width changes. Taking a deeper look and re-trying your suggestions.

Reply:
> make the circle an uneven number width e.g. 13px then you can do left: (50% - 1px) to have true center

I went with this solution 🙏🏽 . @steue had mention `13px` might be too large so went with `11px`.